packages/rs-dpp/src/identity/identity.rs

```diff
@@ -47,33 +81,33 @@ impl Identity {
     }
 
     /// Set Identity public key
-    pub fn set_public_keys(&mut self, pub_key: Vec<IdentityPublicKey>) {
+    pub fn set_public_keys(&mut self, pub_key: BTreeMap<KeyID, IdentityPublicKey>) {
         self.public_keys = pub_key;
     }
 
     /// Get Identity public keys revision
-    pub fn get_public_keys(&self) -> &[IdentityPublicKey] {
+    pub fn get_public_keys(&self) -> &BTreeMap<KeyID, IdentityPublicKey> {
         &self.public_keys
     }
```

Comment:
What is the point to have both setters/getters and the field as public?

Reply:
Completely agree here, we need to go through DPP and remove a lot of useless setters and getters. Just wasn't going to do it in this PR.